src/components/RichTextEditor/utils/parseRawContent.ts

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Descendant } from "slate";
+import { jsx } from "slate-hyperscript";
+import { BlockStyleTypes } from "../renderer/renderBlockStyles";
+import { InlineStyles } from "../renderer/renderInlineStyles";
+
+const BLOCK_MAP: { [key: string]: (el: HTMLElement) => { type: string } } = {
+    A: (el) => ({ type: BlockStyleTypes.Link, url: el.getAttribute("href") }),
+    P: () => ({ type: BlockStyleTypes.Paragraph }),
+    UL: () => ({ type: BlockStyleTypes.UnorderedList }),
+    OL: () => ({ type: BlockStyleTypes.OrderedList }),
+    LI: () => ({ type: BlockStyleTypes.ListItem }),
+};
+
+const STYLE_MAP: { [key: string]: () => { [key: string]: boolean } } = {
+    I: () => ({ [InlineStyles.Italic]: true }),
+    S: () => ({ [InlineStyles.Strikethrough]: true }),
+    STRONG: () => ({ [InlineStyles.Bold]: true }),
+    BOLD: () => ({ [InlineStyles.Bold]: true }),
+    B: () => ({ [InlineStyles.Bold]: true }),
+    U: () => ({ [InlineStyles.Underline]: true }),
+    CODE: () => ({ [InlineStyles.Code]: true }),
+};
+
+export const parseRawValue = (raw?: string): Descendant[] => {
+    let parsedValue: Descendant[] = [{ type: BlockStyleTypes.Paragraph, children: [{ text: "" }] }];
+
+    if (!raw) {
+        return parsedValue;
+    }
+
+    try {
+        parsedValue = JSON.parse(raw);
+    } catch {
+        const parsedDoc = new DOMParser().parseFromString(raw, "text/html");
+        const parsedHtml = deserializeHTML(parsedDoc.body) as Descendant[];
+        if (parsedHtml) {
+            parsedValue = parsedHtml;
+        }
+    }
+
+    return parsedValue;
+};
+
+const deserializeHTML = (element: HTMLElement | ChildNode): Descendant | Descendant[] | string | null => {
```

Comment:
```suggestion
const deserializeHtml = (element: HTMLElement | ChildNode): Descendant | Descendant[] | string | null => {
```
To not have to think if we should uppercase something or not, the Frontify dev team decided to camelCase everything.

Reply:
be0668bd99684b079dea7c7c90eeab2a077acf2f